lib/buffer-serializer.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var configured;
+
+configured = false;
+
+module.exports = (bufferSerializerModule, OtDate) => {
+    if (! configured) {
```

Comment:
Do we do !codeHere or ! codeHere. We have both around the code done by both you and me. I'm for ! codeHere. I like the bang separated from the code/variable looks less like it's all running together.